phygnn/layers/custom_layers.py

```diff
@@ -615,16 +673,21 @@ class SkipConnection(tf.keras.layers.Layer):
     skip start and skip end.
     """
 
-    def __init__(self, name):
+    def __init__(self, name, method='add'):
         """
         Parameters
         ----------
         name : str
             Unique string identifier of the skip connection. The skip endpoint
             should have the same name.
+        method : str
+            Method to use for combining the skip start data and skip end data.
+            Defaults to 'add'. If 'concat' this is applied along the trailing
+            axis
         """
         super().__init__(name=name)
         self._cache = None
+        self._method = method
```

Comment:
I cant remember exactly how this works but i think this is a breaking change for all pretrained sup3r models that will fail on "layer does not have _method attribute". Can you check to see if there's an easy way to fix this using `getattr(self, '_method', 'add')`?

Reply:
Yeah, I could just add that getattr to the call method. You'd prefer having that hack in there over just updating the attrs for old models?

Reply:
open to the conversation, it's really annoying for pretrained models to constantly be incompatible with the latest software, especially for externally published models that are more onerous to update.

Reply:
That's fine, i think we should maybe try to do better about keeping a list of breaking model changes and how to patch though. We could even have functions in phygnn/sup3r that would patch static model files. Maybe too much but it would be nice to know where to go when we hit these issues.

Reply:
Yeah totally agree with this.